lib/mumukit/login.rb

```diff
@@ -4,6 +4,7 @@
 require 'omniauth'
 require 'omniauth-auth0'
 require 'omniauth-saml'
+require 'omniauth-cas'
 require 'omniauth-google-oauth2'
```

Comment:
This change will conflict with your changes @julian-berbel, right?

Reply:
not really. Though cas _should_ be added to the PROVIDERS list in provider.rb and to the default enabled providers for production too

Reply:
also, maybe it would be easier to change default enabled providers for production as PROVIDERS - %w(developer) to avoid having to change this every time we add a new provider

Reply:
Added to providers list